generators/form/templates/formComplex.js.ejs

```diff
@@ -53,14 +43,17 @@ const formFields = {
   bankAccount: 'bankAccount',
   accountType: 'accountType',
   accountNumber: 'accountNumber',
-  bankName: 'bankName',
   routingNumber: 'routingNumber',
   address: 'address',
   email: 'email',
   altEmail: 'altEmail',
   phoneNumber: 'phoneNumber'
 };
 
+function usingDirectDeposit(formData) {
```

Comment:
We typically name these helpers `has[xyz]` so perhaps we could rename this `hasDirectDeposit`.